diff --git a/rtl/include/vpu_pkg.sv b/rtl/include/vpu_pkg.sv
index a179cf5..78dfba5 100644
--- a/rtl/include/vpu_pkg.sv
+++ b/rtl/include/vpu_pkg.sv
@@ -25,7 +25,7 @@ package vpu_pkg;
     typedef logic [31:0] elen_t;    
     
     // Number of bits in a vector register (in each of 32 registers). Min 32. [STATIC]
-    localparam int unsigned VLEN   = 128;	             
+    localparam int unsigned VLEN   = 8192;	             
     
     typedef logic [VLEN-1:0] vlen_t;                    
 
